Extensions/drafts_plus.js

```diff
@@ -23,9 +23,21 @@ XKit.extensions.drafts_plus = new Object({
 					'<div class="hide_overflow" style="color: rgba(255, 255, 255, 0.5) !important; font-weight: bold; padding-left: 10px; padding-top: 8px;">Mass Edit Mode</div>' +
 				'</a>' +
 			'</li>' +
-			'<li class="no_push" style="height: 36px;"><a href="#" onclick="return false;" id="xshrinkposts_button">' +
-				'<div class="hide_overflow" style="color: rgba(255, 255, 255, 0.5) !important; font-weight: bold; padding-left: 10px; padding-top: 8px;">Shrink Posts <div class="count" style="padding-top: 8px;">off</div> </div>' +
-			'</a></li>' +
+			'<li class="no_push" style="height: 36px;">' +
```

Comment:
Can we move `height: 36px` into something like `#drafts_plus_sidebar > li`? It's duplicated for each menu option here.